src/openrct2-ui/windows/Map.cpp

```diff
@@ -860,13 +863,15 @@ static void window_map_paint(rct_window *w, rct_drawpixelinfo *dpi)
                 STR_MAP_TOILET,
             };
 
-            for (sint32 i = 0; i < 8; i++) {
+            for (sint32 i = 0; i < 8; i++)
```

Comment:
Can you replace this with `Util::CountOf(mapLabels)`? You'll probably need to change sint32 accordingly, seeing as CountOf return an unsigned integer.

Reply:
Hmm, you mean a range-based for-loop? The index `i` is used for both `RideKeyColours` and `mapLabels`, though, so I think sticking to the index-based approach is better.

Reply:
In that case, please use an Util::CountOf, rather than a hard coded number. Also make `i` unsigned.

Reply:
Sure, that works. Done.